w-books/src/App/screens/BookDetail/index.js

```diff
@@ -16,7 +15,6 @@ function BookDetail({ match }) {
   let book = BookList.find((element) => { return element.id == match.params.bookId})
```

Comment:
again, move code to `index.js` and ´layout.js`. Do it for all views.

So, do `getSuggestions` and `BookList.find((element) => { return element.id == match.params.bookId})` in `componentWillMount` and store data in state, then pass data to child